src/controllers/update/secure.update.filter.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { NextFunction, Request, Response } from "express";
+
+import * as config from "../../config";
+import { logger } from "../../utils/logger";
+import { ApplicationData } from "../../model";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import { IsSecureRegisterKey } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.SECURE_REGISTER_FILTER_PAGE}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+    const titleExtension: string = "(NOT LIVE)";
+
+    return res.render(config.SECURE_REGISTER_FILTER_PAGE, {
+      backLinkUrl: config.UPDATE_LANDING_PAGE_URL,
+      templateName: config.SECURE_REGISTER_FILTER_PAGE,
+      [IsSecureRegisterKey]: appData[IsSecureRegisterKey],
+      titleExtension
```

Comment:
You might need to extract the logic into a common file to get rid of Sonarqube Duplication failures

Reply:
Separate controllers but they can share common code.

Reply:
I have created a utils file for secure filter which can be used for both register and update